hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/SlashEncodedYearMonthPartitionValueExtractor.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.hudi.hive;
+
+import org.apache.hudi.sync.common.model.PartitionValueExtractor;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * The SlashEncodedYearMonthPartitionValueExtractor class is a custom implementation of the
+ * PartitionValueExtractor interface to support extracting partition values from paths
+ * in the format 'yyyy/mm'.
+ */
+public class SlashEncodedYearMonthPartitionValueExtractor implements PartitionValueExtractor {
+
```

Comment:
How about we name it: SlashedDatetimePartitionValueExtractor

Reply:
Yeah, we can do this. Shall I go-on and make the changes

Reply:
I have made the changes.